docs/how-to-build-a-lexical-model.md

```diff
@@ -0,0 +1,365 @@
+---
+Author: Eddie Antonio Santos
+Date: July 22, 2019
+Keyman Version: 12.0.7-alpha-e58173f
+---
+
+How to build a lexical model
+============================
+
+A **lexical model** or a **dictionary** is a way to enable **predictive text**
+for your keyboard.
+
+Requirements
+------------
+
+You will need to have:
+
+ - [Node.js][] >= 10.0
+ - (Windows only) [Git Bash](https://gitforwindows.org/)
+
+
+[Node.js]: https://nodejs.org/en/
+
+
+Step 1: Get some data
+---------------------
+
+To predict words in your language, a lexical model needs to know the
+words in your language!
+
+Ultimately, the **lexical model compiler** requires a tab-separated
+values (TSV) file, described in the [appendix][]. However, most users will use
+a **spreadsheet** app like [Google Sheets][] or [Microsoft Excel][] to
+create this TSV file.
+
+[Google Sheets]: https://sheets.google.com/
+[Microsoft Excel]: https://products.office.com/en/excel
+
+
+### Example
+
+I have words in my language of choice, SENĆOŦEN.
+
+Here is my list of words, with the count of how many times I've seen the
+word:
+
+| Word  | Count   |
+|-------|---------|
+| TŦE   | 13644   |
+| E     | 9134    |
+| SEN   | 4816    |
+| Ȼ     | 3479    |
+| SW̱    | 2621    |
+| NIȽ   | 2314    |
+| U¸    | 2298    |
+| I¸    | 1988    |
+| ȻSE   | 1925    |
+| I     | 1884    |
+
+I've entered this information into my spreadsheet of choice, [Google
+Sheets][]. I've shared this spreadsheet publicly [here][sencoten-sheet].
+The order of the columns matters:
+
+The first column **MUST** be the words, and the second column **MUST**
+be the counts. Additional columns are ignored. The spreadsheet can also
+be a single column of all of the words in the language.
+
+This is what my word list looks like in Google Sheets:
+
+![The word list, as it appears in Google Sheets](./sencoten-sheets-full.png).
+
+[sencoten-sheet]: https://docs.google.com/spreadsheets/d/10zhIc439BCSSooL_-HeJ6TUHd-ovkiXYcIGe-pHDTSg/edit?usp=sharing
+zsh:1: no matches found: https://docs.google.com/spreadsheets/d/10zhIc439BCSSooL_-HeJ6TUHd-ovkiXYcIGe-pHDTSg/edit?usp=sharing
```

Comment:
Is this supposed to be here? See also next line!

Reply:
haha, I think I keysmashed in Vim here...